### ISSUE TITLE
How to extract moving mean and moving variance from the tensorflow-slim batch norm layer?

### ISSUE THREAD

**Issue by indra215** [tensorflow/models#2541]
I have been trying to extract moving_mean and moving_variance values from the UPDATE_OPS collection for the batchnorm layer in mobilenet. 

```
Tensor("mobilenetv1/mobilenetv1/conv2d_1 depthwise/batchnorm/assignmovingavg:0", shape=(32,), dtype=float32_ref, device=/device:cpu:0)
Tensor("mobilenetv1/mobilenetv1/conv2d_1_depthwise/batchnorm/assignmovingavg_1:0", shape=(32,), dtype=float32_ref, device=/device:cpu:0)
```

However, whenever I try to run the following code to extract the above tensors:

```
variables = tf.get_collection(tf.graphkeys.update_ops)
for var in variables: 
    print(sess.run(var)) 

```
The process hangs and no output can be obtained.
Can anyone please let me know of a solution/alternate method to extract these values?
Thank you!

**Comment by drpngx** [comment]
@aselle who would know about mobilenet?

**Comment by drpngx** [comment]
@derekjchow any idea?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
